src/components/accounts/SetupSeedPhrase.js

```diff
@@ -76,60 +78,74 @@ class SetupSeedPhrase extends Component {
     }
 
     handleCopyPhrase = e => {
-        e.preventDefault()
-        const selection = window.getSelection()
-        selection.selectAllChildren(document.getElementById('seed-phrase'))
-        document.execCommand('copy')
-        selection.removeAllRanges()
+        e.preventDefault();
+        const selection = window.getSelection();
+        selection.selectAllChildren(document.getElementById('seed-phrase'));
+        document.execCommand('copy');
+        selection.removeAllRanges();
+        this.setState({ successSnackbar: true }, () => {
+            setTimeout(() => {
+                this.setState({successSnackbar: false});
+            }, 5500)
```

Comment:
5500 should be a named constant (especially given it's already used in 2 different places)